source/opt/inline_pass.cpp

```diff
@@ -29,8 +29,35 @@ static const int kSpvFunctionCallFunctionId = 2;
 static const int kSpvFunctionCallArgumentId = 3;
 static const int kSpvReturnValueId = 0;
 
+// Constants for OpenCL.DebugInfo.100 extension instructions.
+
+static const uint32_t kDebugInlinedAtOperandInlinedIndex = 6;
+
 namespace spvtools {
 namespace opt {
+namespace {
+
+bool HasInlinedOperand(const Instruction* dbg_inlined_at) {
+  assert(dbg_inlined_at);
+  assert(dbg_inlined_at->GetOpenCL100DebugOpcode() ==
+         OpenCLDebugInfo100DebugInlinedAt);
+  return dbg_inlined_at->NumOperands() > kDebugInlinedAtOperandInlinedIndex;
+}
+
+void UpdateInlinedOperand(Instruction* dbg_inlined_at,
+                          uint32_t recursive_inlined_at_id) {
+  assert(HasInlinedOperand(dbg_inlined_at));
+  dbg_inlined_at->GetOperand(kDebugInlinedAtOperandInlinedIndex).words[0] =
+      recursive_inlined_at_id;
```

Comment:
There are other functions you can use to set the operand: `SetOperand` or `SetInOperand`.  Do not do it this way.  Also this is just a single line.  This should be inlined.  It will be easy enough to read.

Also it updates the "InlinedAt" operand not the "Inlined" operand.  The name should have reflected that.